config/DataDumplog4j.props

```diff
@@ -0,0 +1,17 @@
+# Root logger option
```

Comment:
This has options to either redirect to stdout or to a file. Also, might add more options later for tools. So want to have a separate one.